code/modules/martial_arts/combos/judo/eyepoke.dm

```diff
@@ -0,0 +1,17 @@
+/datum/martial_combo/judo/eyepoke
+	name = "Eye Poke"
+	steps = list(MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Jab your opponent in the eye, damaging and blinding them. Minimally effective on people with eye protection."
+	combo_text_override = "Disarm, Harm"
+
+/datum/martial_combo/judo/eyepoke/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(user.IsKnockedDown())
+		return MARTIAL_COMBO_FAIL
+	target.visible_message("<span class='warning'>[user] jabs [target] in [user.p_their()] eyes!</span>", \
+						"<span class='userdanger'>[user] jabs you in the eyes!</span>")
+	playsound(get_turf(user), 'sound/weapons/whip.ogg', 40, TRUE, -1)
+	target.apply_damage(10, BRUTE)
+	target.AdjustEyeBlurry(50, 0, 30 SECONDS)
+	target.flash_eyes(affect_silicon = FALSE)
```

Comment:
```suggestion
	target.EyeBlind(2 SECONDS)
```

Reply:
Eye-flash damage might be a BIT nasty. This would make it go through eye-pro unless you have a check.

Reply:
Updated to use this in latest!